
Fix the include patch used.

  - Rene Rebe <rene@rocklinux.org>

--- vte-0.10.29/python/vte.c.orig	2003-09-30 17: 42:23.000000000 +0200
+++ vte-0.10.29/python/vte.c	2003-09-30 17:43:33.000000000 +0200
@@ -6,7 +6,7 @@
 
 #line 4 "vte.override"
 #include <Python.h>
-#include <pygtk/pygtk.h>
+#include <pygtk.h>
 #include <pygobject.h>
 #include <gtk/gtk.h>
 #include "../src/vte.h"
--- vte-0.10.29/python/vte.override.orig	2003-09-30 17:42:50.000000000 +0200
+++ vte-0.10.29/python/vte.override	2003-09-30 17:43:31.000000000 +0200
@@ -2,7 +2,7 @@
 %%
 headers
 #include <Python.h>
-#include <pygtk/pygtk.h>
+#include <pygtk.h>
 #include <pygobject.h>
 #include <gtk/gtk.h>
 #include "../src/vte.h"
--- vte-0.10.29/python/vtemodule.c.orig	2003-09-30 17:40:13.000000000 +0200
+++ vte-0.10.29/python/vtemodule.c	2003-09-30 17:40:20.000000000 +0200
@@ -23,7 +23,7 @@
 #endif
 #include <Python.h>
 #include <pygobject.h>
-#include <pygtk/pygtk.h>
+#include <pygtk.h>
 #include "../src/vte.h"
 
 extern void pyvte_register_classes(PyObject * d);
